htdocs/compta/resultat/clientfourn.php

```diff
@@ -301,7 +301,7 @@
 	if (!empty($date_start) && !empty($date_end)) {
 		$sql .= " AND f.doc_date >= '".$db->idate($date_start)."' AND f.doc_date <= '".$db->idate($date_end)."'";
 	}
-	$sql .= " GROUP BY pcg_type DESC";
```

Comment:
I think we must just remove the DESC because the order by is inserted 2 lines later...

Reply:
@eldy you are correct. But the query is not generating correctly that's why MySQL 8.0.30 showing error. Okay I am fixing the code further.